sw/airborne/modules/relativelocalizationfilter/relativelocalizationfilter.c

```diff
@@ -0,0 +1,203 @@
+/*
+ * Copyright (C) Mario Coppola
+ *
+ * This file is part of paparazzi
+ *
+ * paparazzi is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2, or (at your option)
+ * any later version.
+ *
+ * paparazzi is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with paparazzi; see the file COPYING.  If not, see
+ * <http://www.gnu.org/licenses/>.
+ */
+/**
+ * @file "modules/relativelocalizationfilter/relativelocalizationfilter.c"
+ * @author Mario Coppola
+ * Relative Localization Filter for collision avoidance between drones
+ */
+
+#include "relativelocalizationfilter.h"
+#include "subsystems/datalink/telemetry.h"
+#include "modules/multi/traffic_info.h"
+#include "modules/stdma/stdma.h"
+#include "subsystems/datalink/bluegiga.h"
+
+#include "pprzlink/pprz_transport.h"
+#include "subsystems/datalink/datalink.h"
+#include "subsystems/datalink/downlink.h"
+
+#ifndef NUAVS
+#define NUAVS 5				// Maximum expected number of drones
+#endif
+
+#ifndef INS_INT_VEL_ID
+#define INS_INT_VEL_ID ABI_BROADCAST
+#endif
+
+ekf_filter ekf[NUAVS-1]; 	// EKF structure
+btmodel model[NUAVS-1];  	// Bluetooth model structure 
+int IDarray[NUAVS-1]; 		// Array of IDs of other MAVs
+int8_t srcstrength[NUAVS-1];// Source strength
+uint32_t now_ts[NUAVS-1]; 	// Time of last received message from each MAV
+int nf;						// Number of filters registered
+float RSSIarray[NUAVS-1];	// Recorded RSSI values (so they can all be sent)
+
+static abi_event rssi_ev;
+static void bluetoothmsg_cb(uint8_t sender_id __attribute__((unused)), 
```

Comment:
If you define this static you don't need to have a function descriptor and definition, just the definition below is sufficient.